### PR TITLE
Grammar + typo fix

### PR DESCRIPTION
**Pull request by mairaw**
Contributes to #1003

### CODE REVIEW

File: iis/extensions/url-rewrite-module/modifying-http-response-headers.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Modifying HTTP Response Headers"
 author: ruslany
-description: "This section of the documentation applies to the URL Rewrite Module Version 2.0 for IIS 7. This walkthrough will guide you through how to use URL Rewrite Mo..."
+description: "This section of the documentation applies to the URL Rewrite Module Version 2.0 for IIS 7. This walkthrough guides you through how to use URL Rewrite Mo..."
```
Comment (wadepickett) [inline iis/extensions/url-rewrite-module/modifying-http-response-headers.md:4]:
```suggestion
description: "This walkthrough guides you through how to use URL Rewrite Module Version 2.0 for IIS 7."
```

**Review by wadepickett — APPROVED** [review]
Looks good!  Minor suggestion on description meta data.

**Comment by wadepickett** [comment]
I'm not up on this and I am afraid I don't know.  It could have been a product change over time that is not reflected.

**Comment by mairaw** [comment]
No worries, maybe something that @mertozturk80 or @John-Hart could help answer.

**Comment by Rick-Anderson** [comment]
@John-Hart quick question please

**Comment by wadepickett** [comment]
@John-Hart, see Maira's quick question above.

**Comment by John-Hart** [comment]
@sujitnayak can you look into Maira's quick question above about step 4?

**Comment by sujitnayak** [comment]
Yes it should be ORIGINAL_HOST.

**Pull request merged**